src/Strategy/Sha256Strategy.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Humbug
+ *
+ * @category   Humbug
+ * @package    Humbug
+ * @copyright  Copyright (c) 2017 Pádraic Brady (http://blog.astrumfutura.com)
+ * @license    https://github.com/padraic/phar-updater/blob/master/LICENSE New BSD License
+ *
+ * This class is partially patterned after Composer's self-update.
+ */
+
+namespace Humbug\SelfUpdate\Strategy;
+
+use Humbug\SelfUpdate\Updater;
+use Humbug\SelfUpdate\Exception\HttpRequestException;
+use Humbug\SelfUpdate\Exception\InvalidArgumentException;
+
+class Sha256Strategy extends ShaStrategyAbstract
```

Comment:
😛  I like to add it in a PHPStorm template or something to avoid to have to think about it